test-runner/wasi_test_runner/test_case.py

```diff
@@ -14,14 +14,16 @@ class Failure(NamedTuple):
     message: str
 
 
-class Result(NamedTuple):
+class Executed(NamedTuple):
     output: Output
-    is_executed: bool
     failures: List[Failure]
 
-    @property
-    def failed(self) -> bool:
```

Comment:
I'd keep that property for the `Executed` class as it abstracts the implementation detail.